sdk/eventgrid/azure-eventgrid/dev_requirements.txt

```diff
@@ -1 +1,8 @@
--e ../../../tools/azure-sdk-tools
\ No newline at end of file
+-e ../../core/azure-core
+-e ../../../tools/azure-sdk-tools
+-e ../../../tools/azure-devtools
+-e ../../identity/azure-identity
+-e ../../servicebus/azure-servicebus
+-e ../../eventhub/azure-eventhub
+-e ../../storage/azure-storage-queue
```

Comment:
do we need all these as dev requirements? I feel like there are too many dependencies here.

Reply:
They're not required for the code, but only for the samples. Will need to remove as consume samples change.

Reply:
i would suggest removing these if it's just for samples

Reply:
sounds good will remove